File: pset6/mario/mario.py
from cs50 import get_int

# Prompt for height
while True:
    height = get_int("Height: ")
    if(0 < height < 9):
        break

# Print hash pattern
for i in range(height):

    # Initial spaces
    print(" " * (height - 1 - i), end="")

    # Left hashes
    print("#" * (i + 1), end="")

    # Middle space
    print("  ", end="")

    # Right hashes
    print("#" * (i + 1), end="")

    # New line
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Get Height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n > 8 || 1 > n);

    // Print hash pattern
    for (int i = 0; i < n; i++)
    {
        // Initial Spaces
        for (int j = 1 ; j < n - i; j++)
        {
            printf(" ");
        }

        // Left hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        // Gap
        printf("  ");

        // Right hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        // New line
        printf("\n");
    }

}